merge.cpf

```diff
@@ -0,0 +1,8 @@
+[Actions]
+CreateResource:Name=%DB_IRISAPP_DATA,Description="IRISAPP_DATA database"
+CreateDatabase:Name=IRISAPP_DATA,Directory=/usr/irissys/mgr/IRISAPP_DATA
+CreateResource:Name=%DB_IRISAPP_CODE,Description="IRISAPP_CODE database"
+CreateDatabase:Name=IRISAPP_CODE,Directory=/usr/irissys/mgr/IRISAPP_CODE
+CreateNamespace:Name=IRISAPP,Globals=IRISAPP_DATA,Routines=IRISAPP_CODE,Interop=1
+ModifyService:Name=%Service_CallIn,Enabled=1,AutheEnabled=48
+ModifyUser:Name=SuperUser,PasswordHash=a31d24aecc0bfe560a7e45bd913ad27c667dc25a75cbfd358c451bb595b6bd52bd25c82cafaa23ca1dd30b3b4947d12d3bb0ffb2a717df29912b743a281f97c1,0a4c463a2fa1e7542b61aa48800091ab688eb0a14bebf536638f411f5454c9343b9aa6402b4694f0a89b624407a5f43f0a38fc35216bb18aab7dc41ef9f056b1,10000,SHA512
```

Comment:
This doesn't look obvious. Where did you get this PasswordHash? Which password it relates to and how to setup your own?

Reply:
```bash
docker run --rm -it containers.intersystems.com/intersystems/passwordhash:1.1 -algorithm SHA512 -workfactor 10000
```

to create a new password hash